I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 564, in script
    scene party with top
Exception: Parameter 'old_widget' is not known by ATL Transform.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 564, in script
    scene party with top
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/ast.py", line 1678, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/exports.py", line 1758, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/display/core.py", line 2978, in do_with
    return self.interact(trans_pause=True,
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/display/core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/display/core.py", line 3976, in interact_core
    trans = instantiate_transition(None, old_root, layers_root)
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/display/core.py", line 3922, in instantiate_transition
    trans = self.ongoing_transition[layer](
  File "/Users/azein01/Desktop/Capstone/renpy-8.1.3-sdk/renpy/atl.py", line 489, in __call__
    raise Exception('Parameter %r is not known by ATL Transform.' % k)
Exception: Parameter 'old_widget' is not known by ATL Transform.

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.1.3.23091805
Jack Sparrow Case 1.0
Wed Nov 29 00:01:56 2023
